pkg/appmanager/profiles.go

```diff
@@ -191,15 +191,13 @@ func (appMgr *Manager) setServerSslProfile(
 func (appMgr *Manager) handleServerSNIDefaultProfile(
 	rsCfg *ResourceConfig,
 	peerCert string,
+	sKey serviceQueueKey,
 ) {
 	if appMgr.routeConfig.ServerSSL != "" {
 		// User has provided a name
-		profile := ProfileRef{
-			Name:      appMgr.routeConfig.ServerSSL,
-			Partition: rsCfg.Virtual.Partition,
-			Context:   customProfileServer,
-		}
-		rsCfg.Virtual.AddOrUpdateProfile(profile)
+		prof := convertStringToProfileRef(
+			appMgr.routeConfig.ServerSSL, customProfileServer, sKey.Namespace)
```

Comment:
what happened to the Partition previously specified?  Is this handled internally by convertStringToProfileRef?

Reply:
It's handled internally by that function.